languages/js/src/Host.ts

```diff
@@ -89,32 +61,27 @@ export class UserType<Type extends Class<T>, T = any, Query = any> {
  *
  * @internal
  */
-export class Host implements Required<DataFilteringQueryParams> {
+export class Host<Q, R> {
```

Comment:
The new type variables are for the Query returned by `buildQuery` and the Resource returned by `executeQuery`